src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -473,7 +473,14 @@ def write_to_disk(data: pd.DataFrame, path: Path):
         if data[column].dtype.name == "object":
             data[column] = data[column].astype("category")
     if ".hdf" == path.suffix:
-        data.to_hdf(path, "data", format="table", complib="bzip2", complevel=9)
+        data.to_hdf(
+            path,
+            "data",
+            format="table",
+            complib="bzip2",
+            complevel=9,
+            data_columns=data_values.DATA_COLUMNS,
```

Comment:
What happens when these columns aren't present in the `DataFrame`?

Reply:
From my experimentation, nothing. It's a no-op.